src/model/MeterCustom.cpp

```diff
@@ -192,10 +194,23 @@ namespace model {
     return getImpl<detail::MeterCustom_Impl>()->fuelType();
   }
 
-  bool MeterCustom::setFuelType(const std::string& fuelType) {
+  bool MeterCustom::setFuelType(const FuelType& fuelType) {
     return getImpl<detail::MeterCustom_Impl>()->setFuelType(fuelType);
   }
 
+  bool MeterCustom::setFuelType(const std::string& fuelType) {
+    // Special case for custom meters, the only ones that have "Generic" as a possibility
+    if (openstudio::istringEqual("Generic", fuelType)) {
+      return this->setString(OS_Meter_CustomFields::FuelType, "Generic");
+    }
+    try {
+      return setFuelType(FuelType{fuelType});
+    } catch (std::runtime_error& e) {
+      LOG(Debug, e.what());
+      return false;
+    }
+  }
```

Comment:
So here is the `setFuelType(std::string)` one. As you can see, here I handle a special case, otherwise I defer to `setFuelType(FuelType)`

This allows me to call both

`setFuelType("Steam")` and `setFuelType("DistrictHeatingSteam")`

Reply:
It's a special case because "Generic" is not added to DataEnums?

Reply:
Yes. See the comment.

```
    // Special case for custom meters, the only ones that have "Generic" as a possibility
```

I don't think we want to add Generic to FuelType enum just for this object.